tests/chat-rpc-pub-sub/clients/cpp/cli_test.cc

```diff
@@ -0,0 +1,37 @@
+#include "../../../../cpp/wsWebsocketConnection.h"
+
+#include <iostream>
+#include <string>
+
+const char* TOPIC  = "wslink.communication.channel";
+const char* SECRET = "wslink-secret";
+const char* HOST   = "127.0.0.1";
+const char* PORT   = "8080";
+const char* TARGET = "/ws";
+
+using std::cout;
+using std::endl;
+
+int main(int argc, char *argv[])
+{
+  wsWebsocketConnection ws{SECRET};
+  ws.connect(HOST, PORT, TARGET);
+
+  ws.subscribe(TOPIC, 
+      [](const json& json) -> void 
+      {
+        cout << json["id"] << " -> " << json["result"] << endl;
+      });
+
+  std::string line;
+  while (std::cin >> line)
+  {
+    json args = { line.c_str() };
```

Comment:
We may want to exit the loop with a command like `:q` or `exit`. Just an idea.

Reply:
Yes that would make it more usable, currently it just exits with either `Ctrl-D` or `Ctrl-C`.

Reply:
Let's do it as it is a small change and easy to understand.